src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -56,14 +65,14 @@ public JsonNode listProjects(Path directory) throws Exception {
 
                 Path projectPath = directory.toAbsolutePath();
                 String projectStatus = status(project);
-                String allowedActions = allowedActions(project);
+                List<String> allowedActions = allowedActions(project);
                 JsonNode projectProperties = mapper.readTree(project.getProjectPropertiesPath().toFile());
 
                 // add project info to JSON
                 ObjectNode objectNode = mapper.createObjectNode();
                 objectNode.put(PROJECT_PATH, projectPath.toString());
                 objectNode.put(STATUS, projectStatus);
-                objectNode.put(ALLOWED_ACTIONS, allowedActions);
+                objectNode.put(ALLOWED_ACTIONS, String.valueOf(allowedActions));
```

Comment:
i think we want to add it as a list, maybe it needs to be converted to a JsonArray?